venter/tests.py

```diff
@@ -146,12 +146,18 @@ def test_complaint(self):  # pylint: disable=R0915
         # UpVote
         response = self.client.get(url + '1')
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.data['users_up_voted']), 1)
```

Comment:
Why remove this line?

Reply:
There's only one user who has upvoted the complaint. In this case, the test for length of the array is redundant, since that array itself only contains one object. That can be verified by the True/False assertion I've added in for the field 'upvoted'. I can try writing a case for multiple upvotes and verify that the vote count is working properly if you want.

Reply:
No, it isn't redundant. Even if a single object is present, this statement at least verifies that it's an array-like object with one element, which is much (much) better than verifying nothing at all. Sure though, I'll be thankful if you could right a better test.

Reply:
If merging this is urgent, I'll suggesting reverting these two lines for now.